conf/messages.en

```diff
@@ -1094,7 +1094,7 @@ transport.transportMeans.departure.inlandMode.error.required = Select which inla
 transport.transportMeans.departure.identification.title = Which identification do you want to use for this vehicle?
 transport.transportMeans.departure.identification.heading = Which identification do you want to use for this vehicle?
 transport.transportMeans.departure.identification.seaGoingVehicle = Name of a sea-going vehicle
```

Comment:
Could you change this to vessel as well?

Reply:
Can do good spot